raiden/transfer/events.py

```diff
@@ -171,15 +176,15 @@ class ContractSendChannelUpdateTransfer(ContractSendExpirableEvent):
     def __init__(
             self,
             expiration: BlockExpiration,
-            channel_identifier: ChannelID,
-            token_network_identifier: TokenNetworkID,
+            # FIXME: balance_proof below needs to be fully identified, so we could get rid of this
```

Comment:
Is it not? It contains all three fields of the canonical identifier already.

Reply:
I am not sure i understand what is meant by the comment... could you please clarify this a bit more?

Reply:
It was a reply to the fixme comment. The balance proof contains all information of the canonical identifier already, so the canonical_identifier argument is redundant.

Reply:
I realized the FIXME comment was misleading...I changed it and use `balance_proof.canonical_identifier` inside the constructor.